images/scripts/rhel-8-4.bootstrap

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
 
-URL=http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.4/compose/BaseOS/x86_64/images/
+URL=http://download.devel.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.4/compose/BaseOS/x86_64/images/
```

Comment:
Is this going to get updates - like 8.4.1?

Reply:
I am unsure. Other than 8.0.1 (which is currently a nightly) I could not find any z stream releases in download.devel. It would make sense to me that this would get updates like 8.4.1 since it is labeled as 1latest-RHEL-8.4` and not `latest-RHEL-8.4.0`.

Reply:
FTR, in the recently removed rhel-8-3 image (commit 654fa074ceb8965b87b1c0fd37aaa4e2d989b334) we used http://download.devel.redhat.com/released/rhel-8/RHEL-8/8.3.0/BaseOS/x86_64 . But annoyingly this does not exist yet for /8.4.0. But indeed this URL seems better, like a "floating" 8.4.z one. Thanks!